src/Bladeburner/Bladeburner.ts

```diff
@@ -50,10 +50,19 @@ import { GeneralActions } from "./data/GeneralActions";
 import { PlayerObject } from "../PersonObjects/Player/PlayerObject";
 import { Sleeve } from "../PersonObjects/Sleeve/Sleeve";
 import { autoCompleteTypeShorthand } from "./utils/terminalShorthands";
+import { CasualtyReport, KillBladeburnerCasualties, OperationCasualtyOutcome } from "./Actions/Casualties";
 
 export const BladeburnerPromise: PromisePair<number> = { promise: null, resolve: null };
 
-export class Bladeburner {
+export interface OperationTeam {
```

Comment:
It's arguable, but this interface feels like it belongs to Casualties.ts.

Reply:
I wasn't sure. I _think_ this belong to whatever the underlying link between Operations and BlackOperations is. So until we refactor that, I'd be most comfortable sticking it into Action. Action is already aware of "Team Success Bonus", even though it's not aware of team size (That's a discrimanted union on Ops/BOPS). So I suppose out of here is good.

Afterall, purpose of these refactors is to slim down non-essentials from Bladeburner.ts